src/main/java/org/gridsuite/loadflow/server/entities/parameters/LoadFlowParametersEntity.java

```diff
@@ -0,0 +1,209 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.loadflow.server.entities.parameters;
+
+import com.powsybl.iidm.network.Country;
+import com.powsybl.loadflow.LoadFlowParameters;
+import lombok.*;
+
+import jakarta.persistence.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import org.gridsuite.loadflow.server.dto.parameters.LoadFlowParametersInfos;
+import org.gridsuite.loadflow.server.dto.parameters.LoadFlowParametersValues;
+
+/**
+ * @author Ayoub LABIDI <ayoub.labidi at rte-france.com>
+ */
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
To have a constructor without default values
```suggestion
@NoArgsConstructor
@AllArgsConstructor
@RequiredArgsConstructor
```